vmdb/lib/token_manager.rb

```diff
@@ -28,7 +20,28 @@ def self.class_initialize(name = DEFAULT_NS, options = {})
   end
 
   def self.global_token_store(namespace)
-    @global_token_store[namespace] ||= {}
+    @token_caches[namespace] ||= begin
+      if citest_environment?
+        require 'active_support/cache/memory_store'
+        ActiveSupport::Cache::MemoryStore.new(cache_store_options(namespace))
+      else
+        require 'active_support/cache/dalli_store'
+        memcache_server = VMDB::Config.new("vmdb").config[:session][:memcache_server] || "127.0.0.1:11221"
+        ActiveSupport::Cache::DalliStore.new(memcache_server, cache_store_options(namespace))
+      end
+    end
+  end
+
+  def self.cache_store_options(namespace)
+    {
+      :namespace  => "MIQ:TOKENS:#{namespace.upcase}",
+      :threadsafe => true,
+      :expires_in => @config[:token_ttl]
+    }
+  end
+
+  def self.citest_environment?
```

Comment:
I think the ci part is not necessary because this situation applies to local test env as well